README.md

```diff
@@ -94,9 +94,9 @@ Basics (learn more at [baseimage-docker](http://phusion.github.io/baseimage-dock
 
 Language support:
 
- * Ruby 2.3.8, 2.4.6, 2.5.5 and 2.6.3; JRuby 9.2.0.0.
+ * Ruby 2.3.8, 2.4.9, 2.5.7 and 2.6.5; JRuby 9.2.0.0.
    * RVM is used to manage Ruby versions. [Why RVM?](#why_rvm)
-   * 2.5.5 is configured as the default.
+   * 2.5.7 is configured as the default.
```

Comment:
With the code in this pull request the default would actually be 2.6.5. If 2.5.7 is indeed the desired default then image/nginx-passenger.sh has to be changed again (set to 2.5.7, not just revert because it was 2.5.5), and also the failing test updated to check for 2.5.7 instead of 2.6.5. @paveg please let me know if you'd like me to submit a new pull request to change the default ruby to 2.5.7.

Reply:
https://github.com/phusion/passenger-docker/blob/805a395625aa5b9821416f5634b0c74a970ee115/test/customizable_image_spec.rb#L19
Per spec looks like it actually wants latest so perhaps docs need update. Perhaps we need maintainers' opinion here. This spec and others are failing in master branch (i.e. it is not the case that this spec started to fail with your modifications).

Reply:
It's necessary to leave the old one as default.
So, I think it would be better changing README.md document ruby default version